inventory/group_vars/all.yml

```diff
@@ -51,12 +51,15 @@ cluster_name: cluster.local
 # but don't know about that address themselves.
 # access_ip: 1.1.1.1
 
-# Choose network plugin (calico, weave or flannel)
+# Choose network plugin (calico, weave, flannel or fan)
 kube_network_plugin: flannel
 
 # Kubernetes internal network for services, unused block of space.
 kube_service_addresses: 10.233.0.0/18
 
+# example underlay for ubuntu fan network
+#kube_network_fan_underlay: 10.11.0.0/16
```

Comment:
No, that's the `overlay` network (where you select a `/8` for `kube_pods_subnet`), and the `underlay` network is the network that your host is present on/reachable at.

This is the `/16`, that addresses will be mapped back to.